example.ipynb

```diff
@@ -0,0 +1,164 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "**Importing everything you need**"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "from pathlib import Path\n",
+    "\n",
+    "from vprdb.io import export, read_dataset_from_depth, read_dataset_from_point_clouds\n",
```

Comment:
Too many imports are required, it is more usual for me when you import one class, for example `database` and it has all necessary methods for import, export.